classes/controller/FrontController.php

```diff
@@ -860,6 +860,7 @@ public function setMedia()
         ]);
 
         $this->addJS([
+            _THEMES_DIR_.'core.js',
```

Comment:
@djfm @Shudrum  this constant depends on ` __PS_BASE_URI__`: is it an issue ? Is it better to use ``_PS_ROOT_DIR_.'/themes/'`` instead  ?